cars/v1/apm_tracing/base/README.md

```diff
@@ -0,0 +1,24 @@
+This directory contains the keystore configuration for the `apm-tracing` car to enabled APM Tracing.
+
+## Parameters
+
+This configuration allows to set the following parameters with Rally using `--car-params` in combination 
+with `--cars="defaults,apm-tracing"`:
+
+* `apm_secret_token`: A string specifying the APM Tracing secretToken in order to properly send telemetry
```

Comment:
WDYT about mentioning `apm_tracing_server_url` here as well?

Reply:
Yup yup, that was a last minute change, and I forgot to update the docs. I'll add that momentarily